src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraAcl.scala

```diff
@@ -0,0 +1,26 @@
+
+package org.broadinstitute.dsde.agora.server.business
+
+import org.broadinstitute.dsde.agora.server.business.AgoraPermissions._
+
+object AgoraPermissions {
+  val Read = 1 << 0
+  val Write = 1 << 1
+  val Create = 1 << 2
+  val Redact = 1 << 3
+  val Manage = 1 << 4
+  val ReadWrite = Read | Write
+  val All = Read | Write | Create | Redact | Manage
+}
+
+case class AgoraAcl(permissions: Int) {
+  def canRead: Boolean = (permissions & Read) != 0
+
+  def canWrite: Boolean = (permissions & Write) != 0
+
+  def canCreate: Boolean = (permissions & Create) != 0
+
+  def canRedact: Boolean = (permissions & Redact) != 0
+
+  def canManage: Boolean = (permissions & Manage) != 0
+}
```

Comment:
This is neat, but I worry that it's less straightforward than using a Set, especially for creating and modifying acls. I would favor using Sets unless we have a good reason to prefer this way.